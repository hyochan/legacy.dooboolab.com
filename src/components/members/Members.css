@import '../../root.css';

.member {
  /* display: flex; */
  align-items: center;
  flex-direction: column;
  justify-content: center;
  align-self: stretch;
  background-color: #3c96d2;
  & .title {
    margin-top: 80px;
    font-size: 14px;
    line-height: 1;
    text-align: center;
    color: #ffffff;
    @media (--desktop) {
      margin-top: 95px;
      font-size: 18px;
    }
  }

  & .btn {
    margin-top: 60px;
    margin-bottom: 80px;
    width: 200px;
    height: 48px;
    border-radius: 100px;
    font-size: 11px;
    line-height: 1;
    border: solid 2px #ffffff;
    font-weight: bold;
    text-align: center;
    color: #ffffff;
    cursor: pointer;

    @media (--desktop) {
      margin-top: 72px;
      margin-bottom: 115px;
      width: 200px;
      height: 48px;
      border-radius: 100px;
      font-size: 18px;
    }
  }
}
