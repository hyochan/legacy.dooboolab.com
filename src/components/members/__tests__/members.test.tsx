import { render, shallow } from 'enzyme';

import Members from '../';
import React from 'react';
import Store from '../../../stores/appStore';

const props = {
  store: new Store(),
  navigation: {
    navigate: jest.fn(),
  },
  history: {
    push: jest.fn(),
  },
  getString: jest.fn(),
};

// test for the container page in dom
describe('Members page DOM rendering test', () => {
  const page = shallow(<Members {...props} />);

  it('Members page has to be matched to snapshot', () => {
    expect(page).toMatchSnapshot();
  });
});
