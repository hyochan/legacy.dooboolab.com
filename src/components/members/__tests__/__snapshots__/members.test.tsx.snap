// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Members page DOM rendering test Members page has to be matched to snapshot 1`] = `ShallowWrapper {}`;
